doc/api/cli.md

```diff
@@ -178,6 +178,16 @@ added: v8.0.0
 Enable loading native modules compiled with the ABI-stable Node.js API (N-API)
 (experimental).
 
+## `--abort-on-uncaught-exception`
+<!-- YAML
+added: v0.10
+-->
+
+Abort (dump core) when an uncaught exception is thrown.  On most systems this will
```

Comment:
nit: it seems the rest of this file uses just one space after a dot at the start of a sentence.